ambry-network/src/main/java/com.github.ambry.network/http2/Http2NetworkClient.java

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright 2019 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.network.http2;
+
+import com.github.ambry.clustermap.DataNodeId;
+import com.github.ambry.commons.SSLFactory;
+import com.github.ambry.config.Http2ClientConfig;
+import com.github.ambry.network.NetworkClient;
+import com.github.ambry.network.RequestInfo;
+import com.github.ambry.network.ResponseInfo;
+import io.netty.channel.EventLoopGroup;
+import io.netty.channel.nio.NioEventLoopGroup;
+import io.netty.channel.pool.ChannelPool;
+import io.netty.channel.pool.ChannelPoolMap;
+import java.net.InetSocketAddress;
+import java.util.List;
+import java.util.Set;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+/**
+ * A HTTP/2 implementation of {@link NetworkClient}. More implementation details will be added in next PR.
```

Comment:
minor: Add `TODO` before `More implementation details will be added in next PR.`